EIPS/eip-6809.md

```diff
@@ -0,0 +1,683 @@
+---
+eip: 6809
+title: Non-Fungible Key Bound Token
+description: An interface for Non-Fungible Key Bound Tokens, also known as a Non-Fungible **KBT**[^1].
+author: Mihai Onila (@MihaiORO), Nick Zeman (@NickZCZ), Narcis Cotaie (@NarcisCRO)
+discussions-to: https://ethereum-magicians.org/t/non-fungible-key-bound-token-kbt/13625
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-03-31
+requires: 721
+---
+
+A standard interface for Non-Fungible Key Bound Tokens, also known as a Non-Fungible **KBT**[^1].
```

Comment:
@SamWilsn - why was this not flagged by EIPw?
```suggestion
```

Reply:
Hi @Pandapip1, we can remove the 'standard' keyword if that's the issue. Can you guide us on how to proceed with the Source URLs in the footnotes?

Reply:
No, it's that there's text outside of any section. There must not be text outside of any section.